homeassistant/components/litterrobot/entity.py

```diff
@@ -111,3 +112,9 @@ def parse_time_at_default_timezone(time_str: str) -> time | None:
             )
             .timetz()
         )
+
+
+class LitterRobotConfigEntity(LitterRobotControlEntity):
+    """A Litter-Robot entity that can control configuration of the unit."""
+
+    _attr_entity_category = ENTITY_CATEGORY_CONFIG
```

Comment:
Is there a reason for defining this child class rather than just modifying the parent class?

I also wonder if there could be a better name less likely to be confused with ConfigEntry.

Reply:
The parent class is used by the vacuum platform, which should not be a config category.

Reply:
Ok 👍